app/models/status.rb

```diff
@@ -292,18 +292,6 @@ def ordered_media_attachments
     end.take(MEDIA_ATTACHMENTS_LIMIT)
   end
 
-  def replies_count
-    status_stat&.replies_count || 0
```

Comment:
This can never be `nil` due to the implementation of `status_stat`.
https://github.com/guilherme-andrade/mastodon/blob/50a95ecfa24ddea79b8ce01cb2ea4c6a6b555dca/app/models/status.rb#L381-L383

And coalescing to 0 is also unnecessary:
https://github.com/guilherme-andrade/mastodon/blob/50a95ecfa24ddea79b8ce01cb2ea4c6a6b555dca/app/models/status_stat.rb#L25-L35